### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by Haroenv**
Africe --> Africa

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "a.danger_zone": "Danger",
   "a.egypt": "Egypt",
   "a.ethiopia": "Ethiopia",
-  "a.fine": "Fine",
+  "a.fine": "Fine",W
```
Comment (KittyGiraudel) [inline src/locales/en.json]: Not sure what you added here?

File: src/locales/en.json [hunk 2]
```diff
@@ -127,7 +127,7 @@
   "q.very_nice": "It’s very nice.",
   "q.weird_recursive_heaven": "Weird Recursive Heaven",
   "q.what_if_you_let_go": "If you let it go, what does it do?",
-  "q.west_africa": "Is most of West Africe a giant French blob?",
+  "q.west_africa": "Is most of West Africa a giant French blob?",
```
Comment (KittyGiraudel) [inline src/locales/en.json:130]: Good spot!

File: src/locales/en.json (hunk 1 above)
Comment (Haroenv) [inline src/locales/en.json]: Don't know, maybe this is added by the web interface? I'll try to remove it

**Comment by Haroenv** [comment]
Yep, I managed to get rid of it

**Pull request merged**